### DISCUSSION TITLE
Vertical Stacked Cards Layout

### DISCUSSION THREAD

**Discussion by Chaitanya-Raj** [nolimits4web/swiper#6606] [Q&A]
Does Swiper support this kind of layout?

![image](https://user-images.githubusercontent.com/41683865/233348210-9713332c-a82e-43e3-b5aa-def0117d5dbd.png)

How can I achieve this layout?
One way I can do that is by changing the css style of the slides in the effect: card state, but how do I access the change them, as they are being calculated and set on every prev and next movement?

**Comment by reslear** [comment]
or like apple wallet

![image](https://github.com/user-attachments/assets/0779878e-08da-44c5-b6fe-a0e34b7ad388)
